src/main/bash/kobman-env-helpers.sh

```diff
@@ -60,7 +60,7 @@ function __kobman_create_fork
   local environment=$1
   curl -s https://api.github.com/repos/$KOBMAN_NAMESPACE/$environment | grep -q "Not Found"
   if [[ "$?" == "0" ]]; then
-    __kobman_echo_white "Creating a fork under your namespace $KOBMAN_NAMESPACE"
+    __kobman_echo_white "Creating a fork of https://github.com/hyperledgerkochi/$environment under your namespace $KOBMAN_NAMESPACE"
```

Comment:
@asa1997  Please remove namespace hardcoded in github url.

Reply:
$KOBMAN_NAMESPACE is going to be the user's namespace. So thats why it was given like that

Reply:
@asa1997 Use another variable for Upstream or Master Repo Namespace.